crates/pyndakaas/python/pindakaas/encoding.py

```diff
@@ -1,35 +1,63 @@
 from abc import ABC, abstractmethod
 from typing import Iterable, Optional, TypeAlias
 
+# TODO [?] what should we call the inernal database? Database, encoding, formula.. I'm preferring database so it does not overlap with the other terms too much.
 from .pindakaas import CNFInner, Formula, Lit, WCNFInner, Encoder
 
 Constraint: TypeAlias = Formula
 
+RAISES_UNSAT = ":raises Unsatisfiable: If the formula has become unsatisfiable"
```

Comment:
Is this a standard thing people do? This on-the-fly generating of documentation seems bad. Wouldn't this also mean that the interpreter has to do this string concatenation any time the package is loaded?